docs/framework/directory_structure.md

```diff
@@ -10,7 +10,7 @@ The directory structure is organized as follows, only touching parts relevant fo
     - `framework/`: Some basic building blocks (future queue and multiple buffers) and other framework types
     - `parameters/`: Functionality for de/serializing a parameter directory
     - `serialize_hierarchy/`: Traits needed for all types available via Communication
-    - `serialize_hierarchy_derive/`: Derive macro for the `SerializeHierarchy` trait
+    - `serialize_hierarchy_derive/`: Derive macro for the `PathSerialize, PathDeserialize, PathIntrospect` trait
```

Comment:
Yer forgot to change the paths on the left here.

Reply:
done